scripts/common.py

```diff
@@ -41,7 +41,7 @@
 NODE_PATH = os.path.join(OPPIA_TOOLS_DIR, 'node-%s' % NODE_VERSION)
 NODE_MODULES_PATH = os.path.join(CURR_DIR, 'node_modules')
 FRONTEND_DIR = os.path.join(CURR_DIR, 'core', 'templates', 'dev', 'head')
-YARN_PATH = os.path.join(OPPIA_TOOLS_DIR, 'yarn-%s' % YARN_VERSION)
+YARN_PATH = os.path.join(OPPIA_TOOLS_DIR, 'yarn-v%s' % YARN_VERSION)
```

Comment:
Ah -- I meant, can't we change the dir name to yarn-1.17.3? (That's why I suggested a separate PR might be needed.)

Reply:
Sorry I misunderstood, that sounds like out of the scope of this PR. I will create an issue and add todo here.

Reply:
Done